src/content/community/releases/_index.en.md

```diff
@@ -461,6 +521,8 @@ via the [Submariner addon](https://github.com/open-cluster-management/submariner
 * Submariner now supports SRV DNS queries for both ClusterIP and Headless Services. This facilitates Service discovery using
   port name and protocol. For a ClusterIP Service, this resolves to the port number and the domain name. For a Headless Service,
   the name resolves to multiple answers, one for each Pod backing the Service.
+* The CNI detection method in Submariner Operator is now improved to detect the Flannel CNI, even when the Flannel configMap
+  is missing from the cluster.
```

Comment:
This is not related to 0.16.0 release notes

Reply:
@sridhargaddam sent this relnote to the 0.16 branch here: https://github.com/submariner-io/submariner-website/pull/1020

Reply:
Apparently it was inadvertently put in the v0.10.1 section.

Reply:
Oops, thanks for spotting it @maayanf24 and @dfarrell07.

Shall I send an updated PR to fix it in the release-notes-0.16 branch first?

Reply:
Pushed a fix on the release-0.16 branch - https://github.com/submariner-io/submariner-website/pull/1059